helix-term/src/ui/editor.rs

```diff
@@ -815,6 +820,7 @@ impl EditorView {
                 ..
             } => {
                 let (view, doc) = current!(cxt.editor);
+                view.ensure_cursor_in_view(doc, cxt.editor.config.scrolloff);
```

Comment:
this might need to be sorta recursive: even if cursor stays in place, scroll unless cursor is moved away from scrolloff area

Reply:
Btw that bug is caused by the following behavior difference:
1. VTE sends drag events on one-pixel movements that are frequent when holding mouse, even though mouse doesn't move to different character place in terminal
2. Konsole and WezTerm, on the other hand, send drag events only when mouse moves from one character place to another